Api/config/routes.rb

```diff
@@ -1,21 +1,18 @@
 Rails.application.routes.draw do
   scope '/api' do
-  # sessions routes
+    # sessions routes
     post '/login', to: 'sessions#create'
     delete '/logout', to: 'sessions#destroy'
 
-    resources :users, only: [:show, :index] 
+    resources :users, only: %i[show index]
```

Comment:
Please Could you explain me , why  you put `%i` before `[show index]`

Reply:
It's a literal object, `%i` means an array of symbols. So `[:show, :index]` is the same of `%i[show index]`

Reply:
Thanks Kattya for your explanation